templates/community/ionic-blank.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "ionic-blank",
+    "defaultDir": "ionic-blank",
+    "url": "https://luisbytes.dev",
```

Comment:
This site seems unreachable.

Reply:
Hi, if it's my new personal best, I'll lift the site today

Reply:
Hello, the site is up, https://luisbytes.dev/articles/get-started-nuxt-ionic